server/routes.py

```diff
@@ -27,4 +27,15 @@ def tag_by_id(tag_id):
     if request.method == 'DELETE':
         delete_tag_response = database.delete_tag_by_id(conn, tag_id)
         return delete_tag_response
-        
\ No newline at end of file
+
+# subscribers routes the /tags/:id/subscribers endpoint
+@app.route('/tags/<int:tag_id>/subscribers', methods=['GET','POST'])
+def subscribers(tag_id):
+    if request.method == 'GET':
```

Comment:
Is request automatically defined here?

Reply:
[request](https://flask.palletsprojects.com/en/2.3.x/api/#flask.request) is a Flask global object for accessing request data